sdk/policyinsights/Azure.ResourceManager.PolicyInsights/api/Azure.ResourceManager.PolicyInsights.netstandard2.0.cs

```diff
@@ -403,7 +427,7 @@ public partial class PolicyEvent
         public Azure.Core.ResourceIdentifier ResourceId { get { throw null; } }
         public Azure.Core.AzureLocation? ResourceLocation { get { throw null; } }
         public string ResourceTags { get { throw null; } }
-        public string ResourceType { get { throw null; } }
+        public string ResourceTypeString { get { throw null; } }
```

Comment:
Create an issue to track this: https://github.com/Azure/azure-sdk-for-net/issues/31483